app/controllers/admin/conference_controller.rb

```diff
@@ -217,8 +217,8 @@ def conference_params
     def get_users(role_name)
       @role_users = {}
       # Initialize @role variable, so that view can show the role description
-      @role = Role.where(name: role_name, resource: @conference)
-      @role.blank? ? @role_users[role_name] = @role : @role_users[role_name] = @role.first.users
+      @role = Role.where(name: role_name, resource: @conference) || []
```

Comment:
Nop, you are right. I was tyring it with `find_by`, when I added the `|| []` part.